apps/cookbook/src/app/extensions/extensions-landing-page.component.html

```diff
@@ -0,0 +1,37 @@
+<article>
+  <h1>Kirby Extensions for Angular</h1>
+  <p>
+    The Kirby Extensions library is a collection of design guidelines and components that build on
+    top of the core Kirby Design library.
+  </p>
+
+  <p>
+    Components in this library are developed and maintained by product teams, with support from
+    <a
+      class="kirby-external-icon"
+      href="https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby"
+    >
+      Team Kirby
+    </a>
+    .
+  </p>
+
+  <p>
+    The components follow the Kirby Design guidelines, but they differ from the core components by
+    being more product-specific. This means that components in this library are shared across one or
+    more products, and might not be universally useful to the whole ecosystem. Though some
+    components can also become core-candidates and graduate once they are battle-tested and initial
+    experimentation is over.
+  </p>
+
+  <p>
+    Extensions allow for quicker iteration and less focus on generalising behavior, while the test
+    and especially documentation burden is also significantly lower.
```

Comment:
```suggestion
    and the amount of documentation is also significantly lower.
```

Reply:
This is not necessarily true, as the documentation can easily be more verbose. 
But the burden for creating this documentation is significantly lower, as it is very example-driven and more closely coupled to the components code.

Reply:
Aha that makes sense !